src/server/config/auth.js

```diff
@@ -0,0 +1,19 @@
+export const superToken = 'yodaROX';
```

Comment:
As per our offline discussion, this token doesn't seem necessary. Let's just remove this :)